samples/AfterBlazorClientSide/AfterBlazorClientSide.csproj

```diff
@@ -15,6 +15,10 @@
 		</Content>
 	</ItemGroup>
 
+  <ItemGroup>
```

Comment:
You shouldn't be removing any of the samples from the ClientSide version of the demo applications.  Please remove this entry

Reply:
oops. How did that get in there.